app/javascript/components/widgets/widgets/forest-change/tree-loss-plantations/selectors.js

```diff
@@ -93,7 +94,7 @@ export const getSentence = createSelector(
       endYear,
       lossPhrase,
       value: `${format('.3s')(outsideEmissions)}t`,
-      percentage: percentage < 0.1 ? '<0.1%' : `${format('.2r')(percentage)}%`
+      percentage: `${formatNumber({ num: percentage, unit: '%' })}`
```

Comment:
You aren't generating a string here outside of the function so you just need to execute the helper, no need for the ` and $ :)

Reply:
Nice :) Done.